### DISCUSSION TITLE
Prod build not picking up new files in /public?

### DISCUSSION THREAD

**Discussion by PlopTheReal** [vercel/next.js#10886] [Help] [answered]
Sorry if I'm misusing nextjs for this: I got dynamically uploaded images in the /public folder but it looks the server needs a rebuild to deliver them (all works fine in dev). Any idea?

**Version of Next.js**: 9.2.2

**Comment by PlopTheReal** [comment]
So far it looks I've been able to sort this out by doing the following in server.js: 
```
http.createServer((req, res) => {
   const { pathname, query } = parsedUrl
   if (pathname.startsWith('/watheverInPublic')) {
       app.serveStatic(req, res, publicPathPrefix + pathname)
   } else {
       handle(req, res, parsedUrl)
}
})...
```
Is it the right way?

**Reply by PlopTheReal** [reply]
Seems it doesn't work anymore with next 9.4.4. 
Any way to server dynamic files that aren't part of the build?

**Reply by PlopTheReal** [reply]
looks falling back to send() is the way to go.

**Comment by jamesmosier** [comment, marked as answer]
There is no built-in way to do this with Next. In order to have user uploaded photos be served publicly, you would have to use a file storage service (S3, Cloudinary, etc.) or alternatively you would have to store them on your server somewhere and then make the route to that folder public via a proxy/routing configuration.
